Controllers/WeatherForecastController.cs

```diff
@@ -18,15 +18,16 @@ public WeatherForecastController(ILogger<WeatherForecastController> logger)
         _logger = logger;
     }
 
-    [HttpGet(Name = "GetWeatherForecast")]
-    public IEnumerable<WeatherForecast> Get()
+    [HttpGet]
+    [Route("multiplicar")]
+    public double multiplicar(double n1, double n2)
```

Comment:
Metodo deve começar com letra maiúscula 
```suggestion
    public double Multiplicar(double n1, double n2)
```